finetune/scripts/rename_bert_mlm_weights.py

```diff
@@ -0,0 +1,19 @@
+import joblib
```

Comment:
Are we going to start keeping these kinds of scripts in finetune? If so I will add in the ones to port BERT models etc

Reply:
I added them temporarily for purposes of review so you can interpret other code changes w/ these changes in mind.  I think generically we should have done this but things are scattered all over the place right now.